hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/exception/mappers/FallbackExceptionMapper.java

```diff
@@ -30,8 +32,13 @@
 @Provider
 public class FallbackExceptionMapper implements ExceptionMapper<Exception> {
 
+    private static final String UNIQUE_CP = "violates a uniqueness constraint [by___cp]";
```

Comment:
This should be done somewhere on a lower layer of the stack - preferably in Tinkerpop backend or maybe even tinkerpop backend asking the titan provider to translate this. Just to keep the possibility of another backend a real thing ;)